ripe/atlas/tools/probes/__init__.py

```diff
@@ -11,10 +11,6 @@ class Probe(object):
 
     EXPIRE_TIME = 60 * 60 * 24 * 30
 
-    def __init__(self, id, meta_data=None):
-
-        return CProbe(id, meta_data)
-
```

Comment:
PyCharm highlighted it as an error, so I dug into it:

    Cannot return a value from __init__:
    This inspection reports occurrences of return statements with a return value inside
    __init__ methods of classes. A constructor should not return any value.

it also said this:

    This inspection reports discrepancies between declared parameters and
    actual arguments, as well as incorrect arguments (e.g. duplicate named
    arguments) and incorrect argument order. Decorators are analyzed, too.

I think what you may be looking for is `__new__()` or better yet, something like `get()`?